src/Mesh_TEST.cc

```diff
@@ -45,31 +48,46 @@ TEST(DOMMesh, Construction)
 TEST(DOMMesh, MoveConstructor)
 {
   sdf::Mesh mesh;
+  mesh.SetOptimization("convex_decomposition");
   mesh.SetUri("banana");
   mesh.SetSubmesh("watermelon");
   mesh.SetCenterSubmesh(true);
   mesh.SetScale({0.5, 0.6, 0.7});
   mesh.SetFilePath("/pear");
 
+  sdf::ConvexDecomposition convexDecomp;
```

Comment:
```suggestion
  sdf::ConvexDecomposition convexDecomp;
  EXPECT_EQ(nullptr, convexDecomp.Element());
```
improve coverage a little